src/components/ProductCard.jsx

```diff
@@ -50,43 +53,47 @@ const ProductCard = ({
     category: category,
   };
   return (
-    <div className="flex-center">
-      <div className="card card-shadow p-1">
-        <div className="image">
-          <img src={imageUrl} alt={title} />
-        </div>
-        <div className="card-heading-container card-information">
-          <h3 className="font-md font-bold">{storeName}</h3>
-          <h5 className="font-sm font-medium">{title}</h5>
-          <span>₹{price}</span>
-        </div>
-        <div className="card-badge card-badge-position">{category}</div>
-        <div className="favourite-icon favourite-icon-position">
-          <span
-            className="btn btn-link-noBorder"
-            onClick={() => addToWish(wishData)}
+    <div className="product-card">
+      <div className="flex-center">
+        <div className="card card-shadow p-1 product-card-shadow">
+          <div className="image product-img-container">
+            <img src={imageUrl} alt={title} />
+          </div>
+          <div className="card-heading-container card-information">
+            <h3 className="font-md font-bold">{storeName}</h3>
+            <h5 className="font-sm font-medium">{title}</h5>
+            <span>₹{price}</span>
+          </div>
+          <div className="card-badge card-badge-position">{category}</div>
+          <div className="favourite-icon favourite-icon-position">
+            <span
+              className="btn btn-link-noBorder"
+              onClick={() => addToWish(wishData)}
+            >
+              {wishState.wishBasket.find(
+                (item) => item._id === _id
+              ) === undefined ? (
+                <span className="material-icons">favorite_border</span>
+              ) : (
+                <span class="material-icons">favorite</span>
+              )}
+            </span>
+          </div>
+          <div className="product-ratings">
+            <div className="ratings-container">
+              <span className="btn numbered-rating numbered-rating-position">
+                <span>{rating}</span>
+                <span className="material-icons">star</span>
+              </span>
+            </div>
+          </div>
+          <button
+            className="btn btn-success btn-order"
+            onClick={() => addToCart(cartData)}
           >
-            {wishState.wishBasket.find(
-              (item) => item._id === _id
-            ) === undefined ? (
-              <span className="material-icons">favorite_border</span>
-            ) : (
-              <span class="material-icons">favorite</span>
-            )}
-          </span>
-        </div>
-        <div className="ratings-container">
-          <span className="btn numbered-rating numbered-rating-position">
-            <span>{rating}</span>
-            <span className="material-icons">star</span>
-          </span>
+           { 1===2 ? "Order Now" : <Link to="/Cart">Go to cart</Link>} 
```

Comment:
check this condition once

Reply:
it was a placeholder condition, have made the changes